articles/113_ros_time.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Clock and Time
+permalink: articles/clock_and_time.html
+abstract:
+  This article describes the ROS primatives to support programming which can run both in realtime as well as simulated time.
+published: true
+author: '[Tully Foote](https://github.com/tfoote)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Background
+
+Many robotics algorithms inherently rely on timing as well as synchronization.
+To this end we require that nodes running in the ROS network have a synchronized system clock such that they can accurately report timestamps for events.
+
+There are however several use cases where being able to control the progress of the system are important.
```

Comment:
It's not clear how this relates to time. Should it be `cases where being able to control the flow of the system time are important.` or `cases where being able to control the progress of the system by changing the flow of time are important.`?